ludwig/schema/features/text_feature.py

```diff
@@ -66,6 +69,17 @@ class GBMTextInputFeatureConfig(TextInputFeatureConfig):
     )
 
 
+@DeveloperAPI
+@llm_input_config_registry.register(TEXT)
+@ludwig_dataclass
+class LLMTextInputFeatureConfig(TextInputFeatureConfig):
```

Comment:
I may be missing context here, but why parallel cnn?

Reply:
We can change this to anything we'd like. Beneath the surface, we're not actually making use of the encoder at all. Let me see if we can actually skip initializing any encoders. 

The only thing we actually care about here is to to use the tokenizer from the underlying model to preprocess the data since we want to make use of the pretrained vocab

Reply:
We should use passthrough as GBMs do (with similar "ignores it" behavior).

Reply:
Just updated, this is a good catch @w4nderlust @tgaddair